celsos_bookSearch/celsos_bookSearch.xcodeproj/project.pbxproj

```diff
@@ -173,6 +197,8 @@
 			children = (
 				63C0B79E26557BD800EF30AA /* SearchViewController.swift */,
```

Comment:
Usually we do not add the View along with ViewControllers because they have different responsibilities across the app.